Website/admin/Skins/Privacy.ascx.cs

```diff
@@ -70,7 +70,7 @@ protected override void OnLoad(EventArgs e)
                 {
                     hypPrivacy.Text = Localization.GetString("Privacy", Localization.GetResourceFile(this, MyFileName));
                 }
-                hypPrivacy.NavigateUrl = Globals.NavigateURL(PortalSettings.ActiveTab.TabID, "Privacy");
+                hypPrivacy.NavigateUrl = PortalSettings.PrivacyTabId == Null.NullInteger ? Globals.NavigateURL(PortalSettings.ActiveTab.TabID, "Privacy") : Globals.NavigateURL(PortalSettings.PrivacyTabId);
```

Comment:
Based on this check, the prior checks with Null vs -1 should be updated to be consistent

Reply:
The values get read as NullIntegers when null in the DB AFAIK. I'd need to get the tests working to confirm.

Reply:
I agree with Peter, there is no difference for the .Net code, as null values read from the database are converted to Null.NullInteger = -1. This is one of our old burdens, as early versions of .Net did not support nullable data types. From a database perspective, it is preferable to save unset values as Null instead of a fake value, as a) this is more obvious to be not a value and b) you may place a FK on the column, e,g, to Tabs.TabID in this case. 
Note: in general, no value is saved as -1 - however, for roleID, UserID and PortalID, -1 has a specific meaning (All Users, unauthenticatedUser, Host Portal)
During a cleanup process of the database I plan to replace inappropriate -1 values by Null.